templates/lune/store.amp.html

```diff
@@ -17,9 +17,73 @@
 <html ⚡>
   {{> ../../utils/amp-page-head.snip.html}}
   <body class="lune">
-
     {{> ../../components/navbar/navbar.snip.html}}
 
+    <!--
+      To enable the parallax effect on the image, edit store.amp.json:
+        1. Add "amp-fx-parallax" to the list of the extensions
+        2. Change the value of "parallax-amount" to a number (e.g. 0.7)
+    -->
+
+    <!-- STORE PAGE -->
+    {{#store-page}}
```

Comment:
The only issue i have with this PR is the semantics and nested divs 
I left a comment on how you could solve it in your previous PR. Rest if it looks fine.